types/errors.go

```diff
@@ -46,6 +46,10 @@ const (
 	CodeTooManySignatures CodeType = 15
 	CodeGasOverflow       CodeType = 16
 	CodeNoSignatures      CodeType = 17
+	CodeTxInCache         CodeType = 18
```

Comment:
More explanatory, no?

```suggestion
	CodeTxInMempoolCache         CodeType = 18
```